common/option-model.cpp

```diff
@@ -244,33 +257,23 @@ bool option_model::draw_combobox( notifications_model & model,
 
     ImGui::PushItemWidth( new_line ? -1.f : 100.f );
 
-    std::vector< const char * > labels;
-    auto selected = 0, counter = 0;
-    
-    for( auto i = range.min; i <= range.max; i += range.step, counter++ )
-    {
-        if( std::fabs( i - value ) < 0.001f )
-            selected = counter;
-
-        labels.push_back( endpoint->get_option_value_description( opt, i ) );
-    }
+    int selected;
+    std::vector< const char * > labels = get_combo_labels( &selected );
     ImGui::PushStyleColor( ImGuiCol_TextSelectedBg, { 1, 1, 1, 1 } );
 
     try
     {
         int tmp_selected = selected;
```

Comment:
Why do we need tmp_selected, can't we just use selected?
Don't we need to update selected later?

Reply:
You're right, it should be possible to just use `selected`. Doesn't look like `selected` is used after...

Reply:
Done